examples/meta/src/multiclass/lmnn.sg.in

```diff
@@ -0,0 +1,27 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+File f_feats_test = read_csv("@SHOGUN_DATA@/fm_test_real.dat")
+File f_labels_train = read_csv("@SHOGUN_DATA@/label_train_multiclass.dat")
+
+#![create_features]
+Features features_train = create_features(f_feats_train)
+Features features_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![choose_distance]
+int k = 3
+LMNN lmnn(features_train, labels_train, k)
+lmnn.train()
+Distance lmnn_distance = lmnn.get_distance()
```

Comment:
can you already use `get` here?

Reply:
it seems like the Distance object is generated by `get_distance` and there it is not registered with `SG_ADD`

Reply:
mmmh ... ok maybe we can deal with this later....an issue maybe?

Reply:
agreed, it's better to have these integration tests merged in first and then later refactor to make sure we don't break stuff..